frontend/src/pages/ResourceManager/ResourceCard/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Button } from "reactstrap";
+import Modal from "../../../components/Modal";
+
+import Edit from "../../../assets/images/edit.svg";
+import "../styles.scss";
+
+class ResourceCard extends Component {
+  state = {
+    showModal: false
+  };
+
+  toggleModal = () => {
+    this.setState({
+      showModal: !this.state.showModal
```

Comment:
Minor, but can be the source of some hard-to-spot bugs: accessing `this.state` in `setState` won't always work since `setState` is asynchronous (see [here](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous)). Let's opt to use `setState((prevState) => {...})`